### PR TITLE
feat: Add `nv24` skeleton 

### PR DESCRIPTION
**Pull request by kamuik16**
ref #295 
cc @rjan90 
Lmk if I have done something wrong 🙃.

### CODE REVIEW

**Comment by rjan90** [comment]
Hey @kamuik16! Thanks for working on this, it seems like there is a lot of duplicate files here, with just a `=` ending to them, and they should not be there. List of some of these duplicates:

```
builtin/v15/check.go=
builtin/v15/cron/invariants.go=
builtin/v15/datacap/datacap_state.go=
builtin/v15/datacap/invariants.go=
builtin/v15/evm/evm_state.go=
builtin/v15/evm/invariants.go=
builtin/v15/gen/gen.go=
builtin/v15/init/init_actor_state.go=
builtin/v15/init/invariants.go=
builtin/v15/market/cbor_gen.go=
builtin/v15/market/deal_test.go=
builtin/v15/market/invariants.go=
builtin/v15/market/market_state.go=
builtin/v15/market/market_types.go=
builtin/v15/market/methods.go=
builtin/v15/market/set_multimap.go=
builtin/v15/miner/cbor_gen.go=
builtin/v15/miner/deadline_state.go=
builtin/v15/miner/deadlines.go=
builtin/v15/miner/invariants.go=
builtin/v15/miner/methods.go=
builtin/v15/miner/miner_state.go=
builtin/v15/miner/miner_types.go=
builtin/v15/miner/monies.go=
builtin/v15/miner/partition_state.go=
```

**Comment by kamuik16** [comment]
> Hey @kamuik16! Thanks for working on this, it seems like there is a lot of duplicate files here, with just a `=` ending to them, and they should not be there. List of some of these duplicates:
> 
> ```
> builtin/v15/check.go=
> builtin/v15/cron/invariants.go=
> builtin/v15/datacap/datacap_state.go=
> builtin/v15/datacap/invariants.go=
> builtin/v15/evm/evm_state.go=
> builtin/v15/evm/invariants.go=
> builtin/v15/gen/gen.go=
> builtin/v15/init/init_actor_state.go=
> builtin/v15/init/invariants.go=
> builtin/v15/market/cbor_gen.go=
> builtin/v15/market/deal_test.go=
> builtin/v15/market/invariants.go=
> builtin/v15/market/market_state.go=
> builtin/v15/market/market_types.go=
> builtin/v15/market/methods.go=
> builtin/v15/market/set_multimap.go=
> builtin/v15/miner/cbor_gen.go=
> builtin/v15/miner/deadline_state.go=
> builtin/v15/miner/deadlines.go=
> builtin/v15/miner/invariants.go=
> builtin/v15/miner/methods.go=
> builtin/v15/miner/miner_state.go=
> builtin/v15/miner/miner_types.go=
> builtin/v15/miner/monies.go=
> builtin/v15/miner/partition_state.go=
> ```

Fixed in https://github.com/filecoin-project/go-state-types/pull/299/commits/729ea4548e1467a5f3e25b6bcd795265945a5151.

File: builtin/v15/check.go [hunk 1]
```diff
@@ -0,0 +1,385 @@
+package v14
```
Comment (rjan90) [inline builtin/v15/check.go:1]: This should be:

```suggestion
package v15
```

Comment (kamuik16) [reply builtin/v15/check.go:1]: aah, fixing it. 🙂

Comment (kamuik16) [reply builtin/v15/check.go]: > This should be:

Fixed.

**Review by rjan90 — APPROVED** [review]
Looks good now!

**Pull request merged**